app/studio/1.0.x/dec-conf-studio.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Generating Kong Declarative Config with Kong Studio
+---
+
+![Declarative config](https://s3.amazonaws.com/helpscout.net/docs/assets/59e383122c7d3a40f0ed78e2/images/5ea7f7292c7d3a7e9aebbe3f/file-jTMVWOdyOR.gif)
+
+Traditionally, Kong Gateway has always required a database, either PostgreSQL or Cassandra, to store entities such as Routes, Services, and Plugins during run time. The database settings are typically stored in a configuration file called kong.conf.
+
+Kong Gateway 1.1 added the ability to run the Gateway without a database. This ability is called *DB-less mode*. Since not having a database would preclude using the Admin API to create entities, the entities must be declared in another configuration file (YAML or JSON). This file is known as the *declarative configuration*, or *declarative config* for short.
+
+DB-less mode and declarative configuration bring a number of benefits over using a database:
+
+* Reduced lookup latency: all data is local to the cluster’s node.
+* Reduced dependencies: only the Kong Gateway nodes and configuration are required.
+* Single Source of Truth: entities are stored in a single file, that can be source controlled.
+* New deployment models: Kong Gateway can now easily serve as a lightweight node.
+
```

Comment:
I'd move the first gif in this topic here - ideally we don't want to start the topic with a gif with no context.

Reply:
Eh, let's keep it, few users have commented that they like it

Reply:
Poor phrasing - meant move it *to* here :) didn't mean remove!